app/controllers/application_controller/filter.rb

```diff
@@ -47,7 +47,7 @@ def exp_button
     end
 
     if flash_errors?
-      javascript_flash(:flash_div_id => 'adv_search_flash')
+      javascript_flash
```

Comment:
Yeah so this actually breaks the expression editor inside the advanced search.

1. Go to Compute --> Infra --> VM --> actually select the VMs accordion
1. Open the advanced search (arrow next to search field)
1. Add in a field, but do not select what to do with that field (create an un-complete expression)
1. press the "✓" symbol

A flash message should appear in the expresion editor, but it is actually sent to the flash_msg_div below the opened modal.